lib/crawler/cli/schedule.rb

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+#
+
+# frozen_string_literal: true
+
+require 'dry/cli'
+require 'yaml'
+require 'rufus-scheduler'
+
+module Crawler
+  module CLI
+    class Schedule < Dry::CLI::Command
+      desc 'Schedule a recurring crawl of the site'
+
+      argument :crawl_config, required: true, desc: 'Path to crawl config file'
+
+      option :es_config, desc: 'Path to elasticsearch config file'
+
+      def call(crawl_config:, es_config: nil, **)
+        crawl_config = Crawler::CLI::Helpers.load_crawl_config(crawl_config, es_config)
+        if crawl_config.schedule.nil? || crawl_config.schedule[:interval].nil?
+          raise ArgumentError, 'No schedule found in config file'
+        end
+
+        run_schedule(crawl_config.schedule[:interval], crawl_config)
+      end
+
+      def run_schedule(interval, crawl_config)
+        crawl_config.system_logger.info("Schedule initialized with an interval of #{interval}")
```

Comment:
```suggestion
        crawl_config.system_logger.info("Schedule initialized with an schedule of #{interval}")
```

interval implies fixed gaps, whereas schedule sounds more "right" when referencing `CRON`

Reply:
I also reworded it a bit to avoid repeating `schedule` twice.